### DISCUSSION TITLE
Authorizing dns zone in another tenant and subsciption

### DISCUSSION THREAD

**Discussion by recepkarakaya** [polymind-inc/acmebot#378] [Q&A] [answered]
Hello Shibayan,
My function and key vault are in A tenant and B subscription.
My DNS Zone is in C tenant and D subscription.
I have authorized DNS Zone Contributor the function in DNS Zone. And I added the ID of D subsciption to function app settings.
But DNS Zone does not appear in the drop-down list.
Is what I'm trying to do possible?
If possible how can I do?
Thanks for this documents and your answer.

**Comment by shibayan** [comment, marked as answer]
The privilege management feature (Managed Identity, RBAC) that Acmebot uses cannot go beyond Azure AD tenants, so basically you need to deploy it per Azure AD tenant.

It will work fine with different subscriptions for the same tenant.

**Reply by recepkarakaya** [reply]
Thanks for your quick reply.
